src/Home.py

```diff
@@ -51,114 +53,124 @@ def get_df():
 st.markdown("We welcome you to explore the below visualization by zooming into the map directly or by using the filters in the left side menu.")
 
 #map_w = folium.Map()
 #loca = (migrantdf.iloc[0]['lon'],migrantdf.iloc[0]['lat'])
 #custom_marker = folium.Marker(location=loca)
-#custom_marker.add_to(map_w)
+# custom_marker.add_to(map_w)
 #st_map = st_folium(map_w)
 # 1.2 map
 
 #migrantdf['text'] = migrantdf['Cause of Death Abbreviation'] + migrantdf['Total Number of Dead and Missing'].to_string()
 
 fig = go.Figure(data=go.Scattergeo(
-        lon = migrantdf['lon'],
-        lat = migrantdf['lat'],
-        text = migrantdf['Cause of Death Abbreviation'],
-        #hoverlabel=migrantdf['Migration route'],
-        hoverinfo=['text'],
-        mode = 'markers',
-        marker_color = 'rgb(210,30,0)',
-        marker=dict(
-            colorscale='Reds'
-        )
-        ))
+    lon=migrantdf['lon'],
+    lat=migrantdf['lat'],
+    text=migrantdf['Cause of Death Abbreviation'],
+    #hoverlabel=migrantdf['Migration route'],
+    hoverinfo=['text'],
+    mode='markers',
+    marker_color='rgb(210,30,0)',
+    marker=dict(
+        colorscale='Reds'
+    )
+))
 
 fig.update_layout(
-    title = 'Locations of the Reports',
+    title='Locations of the Reports',
     autosize=False,
-    #width=750,
-    #height=400,
-    geo = dict(
+    # width=750,
+    # height=400,
+    geo=dict(
         scope='world',
-        showcountries = True,
-        showocean = True,
-        landcolor = "#f2f2f0",
-        oceancolor = "#cad2d3",
-        showcoastlines = False,
+        showcountries=True,
+        showocean=True,
+        landcolor="#f2f2f0",
+        oceancolor="#cad2d3",
+        showcoastlines=False,
     ),
-    margin={"r":0,"t":0,"l":0,"b":0},
-    )
-    
-
-#st.plotly_chart(fig)
-
-plot = px.scatter_geo(migrantdf, lat='lat',lon='lon',
-                    hover_name='Migration route',
-                    hover_data=['Cause of Death Abbreviation','Total Number of Dead and Missing','Incident year'],
-                    color_discrete_sequence=['red']
-                    )
+    margin={"r": 0, "t": 0, "l": 0, "b": 0},
+)
+
+plot = px.scatter_geo(migrantdf, lat='lat', lon='lon',
+                      hover_name='Migration route',
+                      hover_data={'Cause of Death Abbreviation': True,
+                                  'Total Number of Dead and Missing': True,
+                                  'Incident year': True,
+                                  'lon': False,
+                                  'lat': False
```

Comment:
Adding these 2 lines removed the X Y coordinates showing up in the hover data on the map!

Reply:
Great